scirpy/io/_io.py

```diff
@@ -169,6 +177,13 @@ def _read_10x_vdj_csv(path: Union[str, Path], filtered: bool = True) -> AnnData:
                 is_cell=chain_series["is_cell"],
                 high_confidence=chain_series["high_confidence"],
             )
+
+            # additional cols from CR6 outputs: fwr{1,2,3,4}{,_nt} and cdr{1,2}{,_nt}
+            fwrs = [f"fwr{i}" if i < 5 else f"fwr{i-4}_nt" for i in range(1, 9)]
+            cdrs = [f"cdr{i}" if i < 3 else f"cdr{i-2}_nt" for i in range(1, 5)]
```

Comment:
This code has very unclear semantics... Please either use separate loops for nt and aa or take a look at 
[`itertools.product`](https://docs.python.org/3/library/itertools.html#itertools.product).

Reply:
Update, thanks for the suggestion!